.gitignore

```diff
@@ -9,3 +9,5 @@ Gemfile.lock
 !.sprockets*
 !lib/gollum/public/gollum/stylesheets/_styles.css
 !.github*
+!.env_example
+wiki
```

Comment:
Think this should not be here, right? We're not really expecting the user's wiki to be inside the gollum directory, I think.

Reply:
I have never worked with automated docker image building, so I'm not used to how this works. Your comments got me to think about it more and I have a better understanding.

It only makes sense to include the docker-compose.yml and .env_example files in the repository for building the docker image from the current directory. The purpose of that is to test everything.

That would mean the wiki directory would have to be located somewhere else, like in a folder named wiki. Because this is only for testing, not deploying, one could argue to not include it in the project at all.

If you would like to include the 2 files for testing purposes only, let me know and I'll tailor it for that purpose only.

Otherwise, I should just remove docker-compose.yml, .env_example, and the changes to .gitignore.